src/hooks/useCurrencies.ts

```diff
@@ -88,10 +88,12 @@ export const useCurrencies = () => {
   const convertPriceCurrency = useCallback(
     ({
       price,
-      targetCurrency
+      targetCurrency,
+      amount
     }: {
       price: Price;
       targetCurrency: CurrencyCode;
+      amount: number;
```

Comment:
I am assuming `amount` is supposed to be a multiplier for the Price object to ensure all values are multiplied.
Maybe we could give it a different name, say `multiplier` so it is obvious, and it is not assumed it's the actual monetary amount. 

Just a thought, maybe we could instead, create a helper function say `PriceMapper` or`PriceMultiplier` or whatever... that accepts a function for transforming the monetary fields with for example a multiplification function. We wont need to introduce the multiplier parameter anymore. We could do that outside this function if we need to transform/multiply the price by a multiple

That mapper function could also be used within this converter as well [below](https://github.com/windingtree/win-frontend/blob/99c9b8149b608bc43446f080d86a520a9c1892fa/src/hooks/useCurrencies.ts#L119-L127)